### PR TITLE
Issue 206 deploy mem and cpu logging

### LINKED ISSUE #206: Deploy memory and cpu monitoring to scratch environment
We so far have been feeling our way there but it would be nice to have a more solid understanding of exactly what eks node configuration is optimal for our FOLIO instances.

Right now we have 4 nodes of `t3.xlarge` which is described [here](https://aws.amazon.com/ec2/instance-types/).  Each has 16GB of ram with 4 vCPUs.

`k describe nodes | grep 'Name:\|Allocated' -A 5 | grep 'Name\|memory'` gives:

```
Name:               ip-10-0-129-211.us-west-2.compute.internal
  memory                      5656Mi (38%)  9620Mi (64%)
Name:               ip-10-0-160-151.us-west-2.compute.internal
  memory                      7700Mi (51%)  11216Mi (75%)
Name:               ip-10-0-167-30.us-west-2.compute.internal
  memory                      7670Mi (51%)  9898Mi (66%)
Name:               ip-10-0-96-94.us-west-2.compute.internal
  memory                      8370Mi (56%)  13874Mi (93%)
```

with the last being nearly fully allocated, so we're pretty close to what our pod specs are asking for. What is important to recognize is that this _isn't actually memory_ utilization by the apps, which I think is something very important to know.

To get **that** we need to get memory and cpu memory _utilization monitoring_ going. This can be known for pods and nodes. Each k8s resource has a `/metrics` endpoint that provides info on health. That I believe is enabled by default in EKS -- we just need to hoover it up with something like Prometheus. Once we have memory utilization going I think we'll have a much better idea how "close to the edge" this configuration is.

Although I think as far as k8s memory request limits go we're pretty close. If one node went down could the system operate? Probably not very well because we wouldn't likely have enough RAM for all the pods to start.

Related #174 and #200.

### PR DESCRIPTION
**Pull request by steveellis**
Closes #206.

After doing quite a bit of research the way we want to do this is have both monitoring and logging flow through CloudWatch and bypass Prometheus entirely. The Prometheus we had with rancher was an enhanced one. The one you get via helm with the default config anyway is very basic and nearly unusable. I'm sure with a lot of config you could get it to where it produces something meaningful but we don't have time for that.

The good news is that piping monitoring metrics directly into CloudWatch is easy to set up and has a great dashboard UX. 

This isn't about logging, but from what I see the flow there should also be to pipe logs to CloudWatch and then optionally pipe them from there into ES. In other words CloudWatch is indended for and probably should be our "single pane of glass".

I'm leaving the Prometheus code in this PR since it does work, but I'm removing it from index.ts. Happy to remove it entirely.

The other thing I learned was that there's not much value for us at least for now to customizing the AWS yaml files that do the monitoring deployment. It had the metrics I cared about and a whole lot more. So it's all just a script file and not in pulumi.

### CODE REVIEW

**Review by jwfuller — CHANGES_REQUESTED** [review]
I think we should take out the Prometheus stuff if we aren't using it, maybe cherry-pick it into another branch if you want to keep it around. For the AWS YAML files, I think it is a good idea to make a copy of them in this repo so that we won't be surprised by any future changes or deletion of those files.

**Comment by steveellis** [comment]
Removed prometheus code. Also think we agreed to keep the yaml files as is for now.

File: pulumi/folio/index.ts [hunk 1]
```diff
@@ -9,10 +9,12 @@ import * as kafka from "./kafka";
 import * as postgresql from "./postgresql";
 import * as folio from "./folio";
 import * as util from "./util";
+import * as prometheus from "./prometheus";
```
Comment (jwfuller) [inline pulumi/folio/index.ts:12]: Is this referring to the file that was removed?

**Review by jwfuller — APPROVED** [review]

File: pulumi/folio/index.ts (hunk 1 above)
Comment (steveellis) [reply pulumi/folio/index.ts:12]: Yes let me fix that.

Comment (steveellis) [reply pulumi/folio/index.ts]: There were a few other things I missed. It's gone now.

**Pull request merged**